uninstall-thin-edge_io.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+# Here don't need to remove/purge the tedge_mapper, tedge_agent, and tedge_watchdog packages explicitly,
+# as they will be removed by removing the tedge package.
+Packages=("tedge" "tedge_apt_plugin" "tedge_apama_plugin" "c8y_log_plugin" "c8y_configuration_plugin")
```

Comment:
Variable names that starts with upper case looks quite unusual to my eyes, even though shell doesn't enforce any restrictions as such. Lower case names is the general convention that I've seen.